packages/sdk-utils/src/index.js

```diff
@@ -17,7 +18,8 @@ export {
   fetchPercyDOM,
   postSnapshot,
   postComparison,
-  flushSnapshots
+  flushSnapshots,
+  captureAutomateScreenshot
```

Comment:
This should be `postAutomateScreenshot`

Reply:
https://github.com/percy/cli/pull/1250#discussion_r1213158152